### ISSUE TITLE
Rules surrounding discord server language

### ISSUE THREAD

**Issue by ManonLef** [TheOdinProject/top-meta#314]
| Title | Author | Date |
| --- | --- | --- |
| Add our server language to the rules  | `Manon` | 2024-07-12 |

# Addition to the rules that our discord server is English

## Summary

Add a rule that states we decided on English being the server language and we expect all conversation to happen in English. If needed we can also update the faq on why we made this decision.  

## Motivation

A moderator requested a user to speak English on our server and another earner pointed out it wasn't in the rules. I told them I'd bring it up to the team so here we are. 

## Suggested implementation

As described above 

## Drawbacks

Are there any drawbacks to this proposal? consider the following:

Not that I can think of since we already enforce this, but it will be nicer to have it in the rules as well so we can back this up where needed.

**Comment by ManonLef** [comment]
@TheOdinProject/moderators @TheOdinProject/maintainers I'd like your input.

**Comment by xandora** [comment]
Sounds reasonable, it's the de facto standard we expect anyway. If someone pushed back simply because it's not explicitly stated, then I guess they get a rule for it created.

**Comment by MaoShizhong** [comment]
Agreed. If it's something we enforce, it should be backed by a rule

**Comment by JustWaveThings** [comment]
Agreed. I thought at one point somewhere it is mentioned we communicate in English because all of our resources are in English. I'm not finding that mention now though.

**Comment by Mclilzee** [comment]
This is a good addition to our rules, we always enforced this rule, might as well make it official.

**Comment by rlmoser99** [comment]
I'm not a big fan of having this be a stand-alone rule, but I think we can tie it in with another rule.

> ### Discuss topics that are safe and welcoming for everyone
> ✅ Discuss topics that are relevant to our curriculum or have an off-topic channel.

or

> ### Ask questions in public channels for anyone to answer
> ✅ Ask questions in public channels because everyone in this community shares the responsibility to answer questions and others can learn from reading the conversation. It is unfair to expect specific people to answer your question.

Both of these are close to the "heart" of why we would want to saw that English as the server language.

**Comment by Eduardo06sp** [comment]
I'm in favor of rlmoser's suggestion if it is a mere suggestion/recommendation.

If we want to actually enforce it, we will definitely need to put it in the rules somewhere. 

I saw someone typing in (mostly) Hindi the other day, so I just popped into Google Translate to understand it. Personally, I am in favor of taking rlmoser's approach **without** strictly enforcing unless it becomes an actual problem. 

If I see someone speaking another language, I would rather tell them: "The majority of our users speak English. I highly recommend you communicate in English to increase the chances that you get help for your problem."
as opposed to:
you are not allowed to communicate in languages other than English.

If we want to be open and welcoming to everyone, I think this approach would be more in line with that.

**Comment by bycdiaz** [comment]
I also prefer the idea of this being a suggestion vs a rule.

As we draft that recommendation, I really want us to be explicit about our community being diverse and welcoming to everyone and share reasons why everyone benefits from communicating in a common language. I also ask that we stay away from the specific phrase "English Only" because of the connotations that this phrase can carry.
